brainstormforce/ultimate-addons-for-gutenberg
# Fix: Move dynamic editor CSS to upload directory.

### Description

- Trello - https://trello.com/c/E6bfhMaK/358-uag-warning-for-custom-block-file

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue

### How has this been tested?

- Click on Activate/deactivate the block and check custom-style-blocks.css is created or not in wp-uploads/uag-plugin/custom-style-blocks.css
- Go To the edit page and check by inspecting the page and search  `custom-style-blocks.css` to check it is loading or not.
- Screencast of testing - https://share.getcloudapp.com/E0uAE288

### Checklist:
